lib/backend/model/rule.go

```diff
@@ -82,7 +83,8 @@ type Rule struct {
 }
 
 type HTTPMatch struct {
-	Methods []string `json:"methods,omitempty" validate:"omitempty"`
+	Methods []string         `json:"methods,omitempty" validate:"omitempty"`
+	Paths   []apiv3.HTTPPath `json:"paths,omitempty" validate:"omitempty"`
```

Comment:
Had to pause and think about whether it's ok to import apiv3 from the model package.  I _think_ it is but we need to be careful about introducing cyclic dependencies so typically we don't import the different API levels packages from each other.